src/components/button/_macro.njk

```diff
@@ -9,15 +9,15 @@
       {% else %}
         type="{{ params.type if params.type else ('button' if params.print else 'submit') }}"
       {% endif %}
-      class="btn{% if params.classes %} {{ params.classes }}{% endif %}{% if params.url and not params.hideCaret %} btn--link{% endif %}{% if params.disabled %} btn--disabled{% endif %}{% if params.print %} btn--print u-d-no js-print-btn{% endif %}{% if params.submitType == "loader" %} btn--loader js-loader js-submit-btn{% endif %}{% if params.submitType == "timer" %} js-timer js-submit-btn{% endif %}"
+      class="btn{% if params.classes %} {{ params.classes }}{% endif %}{% if params.url and not params.hideCaret %} btn--link{% endif %}{% if params.disabled %} btn--disabled{% endif %}{% if params.print %} btn--print u-d-no js-print-btn{% endif %}{% if params.exit %} btn--exit{% endif %}{% if params.submitType == "loader" %} btn--loader js-loader js-submit-btn{% endif %}{% if params.submitType == "timer" %} js-timer js-submit-btn{% endif %}"
       {% if params.id %}id="{{ params.id }}"{% endif %}
       {% if params.value %}value="{{ params.value }}"{% endif %}
       {% if params.name %}name="{{ params.name }}"{% endif %}
       {% if params.disabled %} disabled{% endif %}
       {% if params.url and params.newWindow %}target="_blank" rel="noopener"{% endif %}
       {% if params.attributes %}{% for attribute, value in (params.attributes.items() if params.attributes is mapping and params.attributes.items else params.attributes) %}{{attribute}}="{{value}}" {% endfor %}{% endif %}
       >
-      <span class="btn__inner{% if params.innerClasses %} {{ params.innerClasses }}{% endif %}{% if params.newWindow %} icon--external-link{% endif %}">{{- params.html | safe if params.html else params.text -}}</span>
+      <span class="btn__inner{% if params.innerClasses %} {{ params.innerClasses }}{% endif %}{% if params.icon and params.submitType != 'loader' %} icon--{{ params.icon }}{% elif params.newWindow %} icon--external-link {% elif params.exit %} icon--exit {% endif %}">{{- params.html | safe if params.html else params.text -}}</span>
```

Comment:
Think we can lose `params.icon` if we're happy to control button icon set and use the exit icon for save and sign out button